.jenkins/nightly.groovy

```diff
@@ -0,0 +1,64 @@
+pipeline {
+    agent none
+
+    stages {
+
+        stage('Build') {
+            parallel {
+                stage('CUDA-11.4.2') {
+                    agent {
+                        docker {
+                            image 'nvidia/cuda:11.4.2-devel-ubuntu20.04'
+                            label 'NVIDIA_Tesla_V100-PCIE-32GB && nvidia-docker'
+                        }
+                    }
+                    environment {
+                        CTEST_OPTIONS = '--no-compress-output -T Test'
+                        CMAKE_OPTIONS = '-D CMAKE_BUILD_TYPE=Release -D CMAKE_CXX_STANDARD=17 -D CMAKE_CXX_EXTENSIONS=OFF'
```

Comment:
Do we care what CMake build type to use here? I can't think of any benefits of one over the other.

Reply:
I just wanted consistent settings between the configurations.  The actual build type does not matter.